src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -30,6 +30,19 @@ page 7775 "Copilot AI Capabilities"
     {
         area(Content)
         {
+            group(AllowedDataMovementOffInfo)
```

Comment:
Are we planning to have different text for when it's on/off?

Reply:
@darjoo Yes.

On:
![image](https://github.com/user-attachments/assets/0270705f-b767-478b-8dbb-a5e2a61e5adb)


Off:
![image](https://github.com/user-attachments/assets/8db6f27a-02a3-4f47-83c2-e6b03386ae6d)